app/javascript/app/components/country-ndc-sdg-linkages/country-ndc-sdg-linkages.js

```diff
@@ -23,16 +23,20 @@ const mapStateToProps = (state, { match, location }) => {
     data: ndcsSdgsMeta.data,
     activeSector: search.sector
   };
+
   return {
-    fetched: ndcsSdgsData.data[iso],
+    fetched: ndcsSdgsData.data && ndcsSdgsData.data[iso],
```

Comment:
It will always have ndcsSdgsData.data as it is an object, so this is unnecessary.